src/snowcli/cli/nativeapp/version/commands.py

```diff
@@ -0,0 +1,94 @@
+import logging
+from typing import Optional
+
+import typer
+from click import ClickException
+from snowcli.cli.common.cli_global_context import cli_context
+from snowcli.cli.common.decorators import (
+    global_options_with_connection,
+    with_project_definition,
+)
+from snowcli.cli.common.flags import DEFAULT_CONTEXT_SETTINGS
+from snowcli.cli.nativeapp.teardown_processor import NativeAppTeardownProcessor
+from snowcli.cli.nativeapp.version.version_processor import (
+    NativeAppVersionCreateProcessor,
+    NativeAppVersionDropProcessor,
+)
+from snowcli.output.decorators import with_output
+from snowcli.output.types import CommandResult, MessageResult
+
+app = typer.Typer(
+    context_settings=DEFAULT_CONTEXT_SETTINGS,
+    name="version",
+    help="Manage Native App Pkg versions in Snowflake",
+)
+
+log = logging.getLogger(__name__)
+
+
+@app.command()
+@with_output
+@global_options_with_connection
+def create(
+    version: Optional[str] = typer.Argument(
+        None,
+        help=f"""The identifier or 'version string' of the version you would like to create a version and/or patch for.
+        Defaults to undefined, which means the CLI will use the version specified in the manifest.yml.""",
+    ),
+    patch: Optional[str] = typer.Option(
+        None,
+        "--patch",
+        "-p",
+        help=f"""The patch number you would like to create for an existing version.
+        Defaults to undefined if it is not set, which means the CLI will either use the one specified in the manifest.yml,
+        or the framework will determine the patch number itself.""",
+    ),
+    force: Optional[bool] = typer.Option(
+        False,
+        "--force",
+        help="Defaults to unset (False). If set (True), we will implicitly respond “yes” to any prompts that come up.",
+        is_flag=True,
+    ),
+    **options,
+) -> CommandResult:
+    """
+    Creates a new version for an application package, or adds a patch if the version already exists.
```

Comment:
Doesn't it simply create the specified version and patch? I find that description a bit confusing.